src/settings/SettingsScreen.js

```diff
@@ -52,7 +51,6 @@ class SettingsScreen extends PureComponent<Props> {
 
     return (
       <ScrollView style={styles.optionWrapper}>
-        <ModalNavBar canGoBack={false} title="Settings" />
```

Comment:
Oh gosh this was inside the ScrollView! That never made any sense.

Which... oh, I see, this was quite noticeable on iOS.

On Android it could only come up if there was more there than fit vertically. I can make that happen by cranking up the system "Display size" setting plus going into landscape; but otherwise it's pretty unlikely, because we only have a few items on this settings screen.

But iOS has the animation where you can tug the scrollable content around even when there's no scrolling to be done, and the "Settings" header would move right along.